ios/Capacitor/Capacitor/Plugins/Camera.swift

```diff
@@ -101,16 +101,21 @@ public class CAPCameraPlugin : CAPPlugin, UIImagePickerControllerDelegate, UINav
 
   func showPrompt(_ call: CAPPluginCall) {
     // Build the action sheet
-    let alert = UIAlertController(title: "Photo", message: nil, preferredStyle: UIAlertController.Style.actionSheet)
-    alert.addAction(UIAlertAction(title: "From Photos", style: .default, handler: { (action: UIAlertAction) in
+    var promptLabelHeader = call.getString("promptLabelHeader") ?? "Photo"
+    var promptLabelPhoto = call.getString("promptLabelPhoto") ?? "From Photos"
+    var promptLabelPicture = call.getString("promptLabelPicture") ?? "Take Picture"
+    var promptLabelCancel = call.getString("promptLabelCancel") ?? "Cancel"
```

Comment:
```suggestion
    let promptLabelHeader = call.getString("promptLabelHeader") ?? "Photo"
    let promptLabelPhoto = call.getString("promptLabelPhoto") ?? "From Photos"
    let promptLabelPicture = call.getString("promptLabelPicture") ?? "Take Picture"
    let promptLabelCancel = call.getString("promptLabelCancel") ?? "Cancel"
```


use let instead of var as they are not changed and produces a warning